Contentstack.Core/Internals/HttpRequestHandler.cs

```diff
@@ -48,7 +48,7 @@ internal HttpRequestHandler(ContentstackClient contentstackClient)
             var request = (HttpWebRequest)WebRequest.Create(uri);
             request.Method = "GET";
             request.ContentType = "application/json";
-            request.Headers["x-user-agent"]="contentstack-dotnet/2.9.0";
+            request.Headers["x-user-agent"]="contentstack-dotnet/2.10.0";

```

Comment:
Can we put the **version** in config/constant file instead of making changes here